app/views/pages/index.html.erb

```diff
@@ -18,6 +18,7 @@
         <li><%= link_to "Getting started guide", "/docs/tutorials/getting-started" %></li>
         <li><%= link_to "Running builds in Docker containers", "/docs/tutorials/docker-containerized-builds" %></li>
         <li><%= link_to "Autoscaling builds on Elastic CI stack for AWS", "/docs/tutorials/elastic-ci-stack-aws" %></li>
+        <li><%= link_to "Standard and Enterprise Features", "/docs/pages/pipelines/standard-and-enterprise-features" %></li>
```

Comment:
```suggestion
        <li><%= link_to "Standard and Enterprise Features", "/docs/pipelines/standard-and-enterprise-features" %></li>
```

Reply:
@karensawrey this tricky one doesn't need `/pages`. The linter was reporting a 404 though.

Reply:
Oh! I'll add this exception to the style guide. 
Thanks! I Thought the PR was marked as unworthy because of multiple requested changes, didn't realize it was a tricky 404.